scripts/deployment/upload_model_to_huggingface.py

```
@@ -0,0 +1,1220 @@
+    default_labels = [
+        'anxious', 'calm', 'content', 'excited', 'frustrated', 'grateful',
+        'happy', 'hopeful', 'overwhelmed', 'proud', 'sad', 'tired'
+    ]
+
+    print(f"⚠️ Using default emotion labels ({len(default_labels)} classes)")
+    print("   Consider creating emotion_labels.json or setting EMOTION_LABELS environment variable")
+    print("   for better label consistency with your trained model.")
+
+    return default_labels
+
+def prepare_model_for_upload(model_path: str, temp_dir: str) -> dict[str, any]:
```

Ruff (0.12.2)

Too many branches (20 > 12)

(PLR0912)